.github/workflows/radar-release-actions.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [ published ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+      - run: npm run build
+      - run: npm run test
+      - run: npm publish
```

Comment:
might need a way to support `npm publish --tag beta`

Reply:
Yeah, good idea

Reply:
There are two different events we can use for release vs beta release:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Note: The `prereleased` type will not trigger for pre-releases published from draft releases, but the `published` type will trigger. If you want a workflow to run when stable and pre-releases publish, subscribe to `published` instead of `released` and `prereleased`.

(and then when creating the release, you check the box that says "this is a pre-release"

Reply:
Basically, `released` vs `prereleased`

Reply:
Ah nice, will use that.